### PR TITLE
#3860 UI bugs fixes

### PR DESCRIPTION
**Pull request by drakkein**
This PR contains
- Fixes broken layout for Metamask login page on mobile devices
- Adds test for ETO Individuals page (bug fixed by @arjunu )
- Fixes `Link New Account` button position in Bank Account component
- Fixes ETH send amount field error message
- Fixes token symbol letter size in Token Transfer modal
- Adds margin to Token Transfer modal error message
- Fixes social media icon opening modal on click event
- Fixes single file upload component allowing to remove uploaded item while disabled
- Fixes ISHA upload long file name breaking modal size
- Fixes KYC progress bar position on mobile devices
- Fixes size of icon in access wallet modal
- Fixes inline button position (ICBM `Click here` on login page)
- Fixes tooltip icon position (TEMP solution, gonna be replaced by new tooltip)

Things that are not in this PR
- `Video` header was fixed by @AlexeyTuboltsev.
- Tooltips in table has been removed during portfolio redesign

### CODE REVIEW

File: packages/web/app/components/shared/SingleFileUpload.tsx [hunk 1]
```diff
@@ -33,15 +33,16 @@ interface IProps {
 
 const SingleFileUploadUploaded: React.FunctionComponent<Pick<
   IProps,
-  "file" | "onRemove" | "onDownload" | "data-test-id"
->> = ({ file, onRemove, onDownload, "data-test-id": dataTestId }) => (
+  "file" | "onRemove" | "onDownload" | "data-test-id" | "disabled"
```
Comment (arjunu) [inline packages/web/app/components/shared/SingleFileUpload.tsx]: here `disabled` could mean upload disabled, so should the prop be called  `disableRemove`?

File: packages/web/intl/locales/en-en.json [hunk 2]
```diff
@@ -297,7 +297,7 @@
   "eto.documents.reservation-and-acquisition-agreement": "Reservation and Acquisition Agreement",
   "eto.documents.reservation-and-acquisition-agreement-template": "Reservation and Acquisition Agreement Template",
   "eto.documents.signed-investment-and-shareholder-agreement": "Signed Investment and Shareholders Agreement",
-  "eto.documents.signed-investment-and-shareholder-agreement.hashes-dont-match": "There is a problem with the ISHA you uploaded.<br>File {uploadedFileName} doesn't match the copy the Issuer uploaded. Please try again.",
+  "eto.documents.signed-investment-and-shareholder-agreement.hashes-dont-match": "There is a problem with the ISHA you uploaded.{breakLine}File {uploadedFileName} doesn't match the copy the Issuer uploaded. Please try again.",
```
Comment (arjunu) [inline packages/web/intl/locales/en-en.json]: where is `{breakLine}` coming from? 

the usage is 
```
 <FormattedHTMLMessage
            tagName="span"
            id="eto.documents.signed-investment-and-shareholder-agreement.hashes-dont-match"
            values={{ uploadedFileName }}
          />
```

Comment (drakkein) [reply packages/web/intl/locales/en-en.json]: This is passed as another value `breakLine: <br /`, pretty common way in our app.

Comment (arjunu) [reply packages/web/intl/locales/en-en.json]: but why is my question, since it's a constant and it will not change.

Comment (drakkein) [reply packages/web/intl/locales/en-en.json]: I had to add some additional styles just to `uploadedFileName` so it was transformed into span. Unfortunately `FormattedHTMLMessage` has problem with rendering `React.Element`, we will just see `[object]`. In that case I had to make it `FormattedMessage` and pass `<br />` as value.

**Comment by Moejoe90** [comment]
will temporary close

**Pull request closed without merging**